### ISSUE TITLE
Making it work

### ISSUE THREAD

**Issue by prakhar2611** [prakhar2611/lexical-app#2]
Okay you have done with importing the provided code from their page, but now what, How to make this work?? 
Expectation that we had from this was to create doc and save the doc so, we end up having our own editor + docs for _Ideas_

**Comment by prakhar2611** [comment]
So lets start with **requirements** 

Get familiar with editor and its playground, the way it has been implemented, how does it works 
`Fundamentals`

- lexical composer 
- lexical state
- initial state (one time loader only)
- get editor state 
- update editor state

**Comment by prakhar2611** [comment]
After Understanding few thing about lexical, I need to think of how I should save a file from the the given content, the official documentation already had a code of how to get the on change listener plugin and get the content of the editor and saving in json format to send to backend, **See its all out there**. So I tried and it worked. 

Now as I got the content format so I switched to my backend. As this was my weekend project (picked yesterday) and I need a backend to serve the files over my db (currently running on the raspi server locally on my home network ! Read about how we setup raspi and make it our usage **_here_** ) Next issue is about that.
